addon/templates/components/polaris-data-table/cell.hbs

```diff
@@ -1,8 +1,22 @@
 {{#if presentational}}
   {{#if header}}
-    <th aria-hidden="true" role="presentation" id={{cellElementId}} class={{cellClassNames}} style={{style}}></th>
+    <th
+      aria-hidden="true"
+      role="presentation"
+      id={{cellElementId}}
+      class={{cellClassNames}}
+      style={{style}}
+    >
+    </th>
```

Comment:
Are you aware of any reason for this not being made self-closing like a couple of the `div`s in this PR?

Reply:
no reason....just missed these